comms/src/control_service/mod.rs

```diff
@@ -25,6 +25,25 @@
 //! The control service listens on the configured address for [RequestConnection] messages
 //! and decides whether to connect to the requested address.
 //!
+//! The protocol for establishing a peer connection is as follows:
+//!
+//! Node A                  Node B
+//!  +                       +
+//!  |  ReqConn(n_id, addr)  |
+//!  | +-------------------> |
+//!  |                       | Create Inbound PeerConnection
+//!  |  Accept(c_pk, addr)   |
```

Comment:
I take it, this is an or situation, and you dont send accept them recjectm then try to connect?

Reply:
Yup one or the other happens - I'll make mention of that in the docs